nusight2/src/client/components/three/builders.ts

```diff
@@ -143,7 +143,11 @@ type MeshOpts = Object3DOpts & {
 }
 
 export const mesh = createUpdatableComputed(
-  (opts: MeshOpts) => new Mesh(opts.geometry, opts.material),
+  (opts: MeshOpts) => {
+    const mesh = new Mesh(opts.geometry, opts.material)
+    updateObject3D(mesh, opts)
```

Comment:
This already gets called https://github.com/NUbots/NUbots/blob/f9cd04276825069e40c8842ea7ae6ec9a661620a/nusight2/src/client/components/three/create_updatable_computed.ts#L25